adapters/pom.xml

```diff
@@ -82,6 +82,10 @@
       <groupId>io.opentracing</groupId>
       <artifactId>opentracing-noop</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.wildfly.security</groupId>
+      <artifactId>wildfly-elytron-password-impl</artifactId>
```

Comment:
out of curiosity: which component uses/requires this dependency?

Reply:
I have not had a deep look into this. But it looked like it is the "native" Kafka client. When I configured the Kafka client to use the SASL mechanism _SCRAM-SHA-256_, the authentication failed with an exception related to Wildfly and password checking. Adding this dependency did the trick...

Reply:
@sophokles73 Is this sufficient from your POV? Or should we investigate further?

Reply:
No, all good